src/agents_gbtree.erl

```diff
@@ -2,49 +2,48 @@
 %%% @author Daniel Grzonka
 %%% @copyright (C) 2014, <COMPANY>
 %%% @doc
-%%% Ants gb_trees based approach
+%%% Agents gb_trees based approach
 %%% @end
 %%% Created : 07. lis 2014 15:56
 %%%-------------------------------------------------------------------
--module(ants_gbt).
+-module(agents_gbtree).
 -author("Daniel").
 
 %% API
--export([create_ants/4, partition/3, get_agent/3, update_agent/4, group_by/1]).
+-export([create_agents/3, partition/3, get_agent/3, update_agent/4, group_by/1]).
```

Comment:
Line 13 is too long: -export([create_agents/3, partition/3, get_agent/3, update_agent/4, group_by/1])..